src/database/index.tsx

```diff
@@ -8,12 +8,15 @@ import createAuth0Client from "@auth0/auth0-spa-js";
 
 import config from "../auth/auth_config.json";
 import { EpidemicModelPersistent } from "../impact-dashboard/EpidemicModelContext";
+import { Facility } from "../page-multi-facility/types";
```

Comment:
For the time being, I just continued my work in the folder that was already in progress (i.e. "page-multi-facility"), but this folder should likely be renamed in the future to something like "multi-facility-impact-dashboard" so that it doesn't appear that backend code depends on frontend code.